### PR TITLE
Add French translations

### PR DESCRIPTION
**Pull request by MetotoSakamoto**
Add French Translation

### CODE REVIEW

File: po/fr.po [hunk 1]
```diff
@@ -0,0 +1,659 @@
...
+msgstr "Synchroniser l'entrée"
+
+msgid "Copy"
+msgstr "Copier"
+
+msgid "Delete key generates"
+msgstr "Delete key generates"
+
+msgid "Edit Profile"
+msgstr "Editer Profile"
```
Comment (Vinibo) [inline po/fr.po]: Should be "Modifier" instead of Editter

File: po/fr.po [hunk 2]
```diff
@@ -0,0 +1,659 @@
...
+
+msgid "Blue"
+msgstr "Bleu"
+
+#, c-format
+msgid "Select %s Light Color"
+msgstr "Sélectionner %s Couleur claire"
+
+msgid "Preferences"
+msgstr "Preferences"
```
Comment (Vinibo) [inline po/fr.po]: "Préférences"

File: po/fr.po [hunk 3]
```diff
@@ -0,0 +1,659 @@
...
+msgstr "Sombre"
+
+msgid "Save"
+msgstr "Sauvegarder"
+
+msgid "Terminix"
+msgstr "Terminix"
+
+msgid "About"
+msgstr "A propos"
```
Comment (bil-elmoussaoui) [inline po/fr.po]: should be `À propos` instead of `A propos`

**Comment by gnunn1** [comment]
Can you make those suggested changes and re-submit.

**Comment by gnunn1** [comment]
sorry, should not have closed it, for got if you just update the files it comes through automatically,

**Comment by MetotoSakamoto** [comment]
Changes done, it's fun. : D
I discover github.

File: po/fr.po [hunk 4]
```diff
@@ -0,0 +1,659 @@
...
+msgstr "Sauvegarder"
+
+msgid "Terminix"
+msgstr "Terminix"
+
+msgid "About"
+msgstr "À propos"
+
+msgid "rows"
+msgstr "rows"
```
Comment (bil-elmoussaoui) [inline po/fr.po]: need to be translated

File: po/fr.po [hunk 5]
```diff
@@ -0,0 +1,659 @@
...
+
+msgid "Match entire word only"
+msgstr "Match entire word only"
```
Comment (bil-elmoussaoui) [inline po/fr.po]: need to be translated

File: po/fr.po [hunk 6]
```diff
@@ -0,0 +1,659 @@
...
+
+msgid "IBeam"
+msgstr "IBeam"
+
+msgid "Run command as a login shell"
+msgstr "Run command as a login shell"
```
Comment (bil-elmoussaoui) [inline po/fr.po]: need to be translated

File: po/fr.po [hunk 7]
```diff
@@ -0,0 +1,659 @@
...
+
+msgid "Exit the terminal"
+msgstr "Quitter le terminal"
+
+msgid "Restart the command"
+msgstr "Redémarrer la commande"
+
+msgid "Choose A Terminal Font"
+msgstr "Choose A Terminal Font"
```
Comment (bil-elmoussaoui) [inline po/fr.po]: Should be `Choisir une police`

File: po/fr.po [hunk 8]
```diff
@@ -0,0 +1,659 @@
...
+msgstr "Curseur"
+
+msgid "Encoding"
+msgstr "Codage"
+
+msgid "Global"
+msgstr "Global"
+
+msgid "Set the starting profile"
+msgstr "Set the starting profile"
```
Comment (bil-elmoussaoui) [inline po/fr.po]: Shoud be `Définir le profile du démarrage` (i guess :p)

**Comment by gnunn1** [comment]
The keyboard categories and shortcuts at the end should appear in normal human text, i.e. title-case and without the dashes, see the en.po for an example. When you get a chance can you make that change?

**Pull request merged**